src/views/Log.vue

```diff
@@ -429,7 +436,18 @@ export default {
         id: this.id ?? undefined, // (do not trigger the callback on null ⇄ undefined)
         file: this.file ?? undefined
       }),
-      async ({ id }, old) => {
+      async ({ id, file }, old) => {
+        // update the widget tab caption when the id or file change
+        if (this.widgetID) {
+          const prefix = this.relativeID ? `${this.relativeID} – ` : ''
+          eventBus.emit(
+            `lumino:update-tab:${this.widgetID}`,
+            {
+              title: this.jobLog ? 'Log – Job' : 'Log – Workflow',
```

Comment:
Consider using fewer and narrower chars?
```suggestion
              title: this.jobLog ? 'Log: Job' : 'Log: Workflow',
```